packages/user-interface/src/components/BackLink.tsx

```diff
@@ -0,0 +1,39 @@
+import { useContext } from "react";
+import { FormattedMessage } from "react-intl";
+import { LocaleContext } from "@nl-portal/nl-portal-localization";
+import { Link } from "@gemeente-denhaag/link";
+import { ChevronLeftIcon } from "@gemeente-denhaag/icons";
+import styles from "./BackLink.module.scss";
+import { PortalLink } from "..";
+import { useNavigate } from "react-router-dom";
+
+export interface BackLinkProps {
+  routePath?: string;
+  text?: string;
+}
+
+const BackLink = ({ routePath, text }: BackLinkProps) => {
+  const { hrefLang } = useContext(LocaleContext);
+  const navigate = useNavigate();
+
+  // TODO: in the future, the Link should be a button that looks like a link if the action is a onClick.
```

Comment:
Deze is wel al beschikbaar als je wil. Heb die op de CasePage gebruikt. Als routePath undefined is kunnen je die gewoon return.

className="utrecht-button-link utrecht-button-link--html-a"

Reply:
Dat is de omgekeerde variant, maar ik heb inmiddels een link-button gemaakt/gebruikt van Utrecht die ik ga implementeren.